acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/KeyLoadAndSignAcceptanceTest.java

```diff
@@ -51,7 +53,7 @@
   @TempDir Path testDirectory;
 
   @ParameterizedTest
-  @ValueSource(strings = {"/signer/block", "/signer/attestation"})
+  @ValueSource(strings = {"/signer/block", "/signer/attestation", "/signer/randao_reveal"})
```

Comment:
is the randao endpoint a different PR?

Reply:
Maybe it should be? But since it has the same payload I thought the changes would be small enough to include in this PR. Happy to split it out though if you think that is better.

Reply:
Based further discussion we have decided to leave this in since it is small